CITATION.cff

```diff
@@ -1,290 +1,431 @@
 title: 'Gammapy: Python toolbox for gamma-ray astronomy'
 abstract: 'Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves,
   from event lists and instrument response information; it can also determine the
   position, morphology and spectra of gamma-ray sources. It is used to analyze data
-  from H.E.S.S., Fermi-LAT, and the Cherenkov Telescope Array (CTA).'
+  from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).'
+type: software
+message: "If you use this software, please cite it using the metadata from this file."
+url: "https://gammapy.org/"
 doi: 10.5281/zenodo.4701488
+preferred-citation:
+  authors:
+  - family-names: "Deil"
+    given-names: "Christoph"
+  - family-names: "et al."
+  title: "Gammapy - A prototype for the CTA science tools"
+  doi: 10.48550/ARXIV.1709.01751
+  url: "https://arxiv.org/abs/1709.01751"
+  type: conference-paper
+  year: 2017
+  conference:
+    name: "Proc. 35th ICRC, Busan, South Korea, PoS(ICRC2017)766"
+  publisher:
+    name: "arXiv"
+keywords:
+- Astronomy
+- "Gamma-rays"
+- "Data analysis"
+version: "1.0"
+date-released: 2022-09-15
+repository-code: "https://github.com/gammapy/gammapy"
+license: BSD-3-Clause
+contact:
+- email: GAMMAPY-COORDINATION-L@IN2P3.FR
+  name: "Coordination committee of the Gammapy project"
 authors:
-- given-names: Axel
-  family-names: Donath
-  affiliation: Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St.,
-    02138 Cambridge MA, US
+- family-names: Acero
+  given-names: Fabio
+  email: fabio.acero@cea.fr
+  orcid: https://orcid.org/0000-0002-6606-2816
+  affiliation: AIM (DAp)
+  city: Saclay, FR
+- family-names: Armstrong
+  given-names: Thomas
+  email: thomas.armstrong@physics.ox.ac.uk
+  orcid: https://orcid.org/0000-0001-5067-2620
+  affiliation: CPPM
+  city: Marseille, FR
+- family-names: Arribas
+  given-names: Manuel Paz
+  orcid: https://orcid.org/0000-0002-4404-9899
+- family-names: Bigas
+  given-names: Oscar Blanch
+  email: blanch@ifae.es
+  orcid: https://orcid.org/0000-0002-8380-1633
+  affiliation: IFAE
+  city: Barcelona, SP
+- family-names: de Bony de Lavergne
+  given-names: Mathieu
+  email: lavergne@lapp.in2p3.fr
+  orcid: https://orcid.org/0000-0002-4650-1666
+  affiliation: Université Savoie Mont Blanc, CNRS, Laboratoire d’Annecy de Physique des Particules - IN2P3
+  city: Annecy, FR
+- family-names: Bose
+  given-names: Debanjan
+  email: debaice@hotmail.com
+  orcid: https://orcid.org/0000-0003-1071-5854
+- family-names: Brügge
+  given-names: Kai
+  email: kai.bruegge@tu-dortmund.de
+  affiliation: Technical University of Dortmund
+  city: Dortmund, DE
+- family-names: Buchner
+  given-names: Johannes
+  email: johannes.buchner.acad@gmx.com
+  orcid: https://orcid.org/0000-0003-0426-6634
+  affiliation: Max Planck Institute for extraterrestrial Physics
+  city: Garching, DE
+- family-names: Cabot
+  given-names: Arnau Aguasca
```

Comment:
It should be the following:
family-names: Aguasca-Cabot
given-names: Arnau
Thanks!

Reply:
Thanks! Is your affiliation correct?

Reply:
I would prefer to have this one: "Departament de Física Quàntica i Astrofísica, Institut de Ciències del Cosmos, Universitat de Barcelona, IEEC-UB".
Thank you very much.